src/CreatorToken.sol

```diff
@@ -184,16 +207,23 @@ contract CreatorToken is ERC721 {
     view
     returns (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee)
   {
-    _tokenPrice = BONDING_CURVE.priceForTokenNumber((totalSupply + 1) - _preMintOffset());
-    (_creatorFee, _adminFee) = calculateFees(_tokenPrice);
+    (_tokenPrice, _creatorFee, _adminFee) = priceForTokenId(totalSupply + 1);
   }
 
   function nextSellPrice()
     public
     view
     returns (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee)
   {
-    _tokenPrice = BONDING_CURVE.priceForTokenNumber(totalSupply - _preMintOffset());
+    (_tokenPrice, _creatorFee, _adminFee) = priceForTokenId(totalSupply);
+  }
+
+  function priceForTokenId(uint256 _tokenId)
+    public
+    view
+    returns (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee)
+  {
+    _tokenPrice = BONDING_CURVE.priceForTokenNumber(_tokenId - _preMintOffset());
```

Comment:
Our math in this method is wrong because it does *not* account for burns. It assumes that `_tokenId` and `_totalSupply` are always the same.

The first thing we should do is write a test case that (currently) fails because of this bug. Then we should fix it!

Reply:
I thought I had it, but I think it's returning the wrong sell prices, could you check my math [43728e6](https://github.com/showtime-xyz/creator-tokens/pull/29/commits/43728e6415b85c2b58014ab0e5adabb84a43efa5)

Reply:
Maybe I'm just going at it wrong, cause there are going to be no prices for Ids that have been burnt. Or at least there shouldn't be.